day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,16 @@
+numbers = [*1..100]
+numbers.map! { |num|
```

Comment:
What was the reason behind choosing the `.map!` method here?  That method is replacing the current `|num|` with the text output in the array, effectually mutating the originally `numbers` array which is usually something we want to avoid doing when programming.  Have you thought about trying to solve this with a `for` loop?